packages/frontend-core/src/components/grid/overlays/MenuOverlay.svelte

```diff
@@ -161,6 +168,15 @@
           >
             Delete row
           </MenuItem>
+          <MenuItem
```

Comment:
Does this need to be feature flagged?

Reply:
It does - added support to the PR for that!